src/lib.rs

```diff
@@ -1098,6 +1098,21 @@ mod test {
     }
 
     #[test]
+    #[should_panic(expected = "Unexpected value")]
```

Comment:
Rather than calling `unwrap` on the result of `from_str`, let's assert that the result `is_err()` instead.